test/fixtures/questions.yml

```diff
@@ -1,6 +1,7 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 one:
   content: What is your gender?
+  status: draft
```

Comment:
I swear this was needed to fix something, and now I can't figure out what it was.